translations/puddletag_de.ts

```diff
@@ -3669,17 +3669,17 @@ Tatsächlich: %2</translation>
     <message>
         <location filename="../puddlestuff/webdb.py" line="552"/>
         <source>Retrying search with %s</source>
-        <translation type="unfinished"></translation>
+        <translation>Wiederhole die Suche mit %s</translation>
```

Comment:
Maybe: Erneuter Versuch der Suche mit ...

It's "Retrying" - present continuous ... Wiederhole is in the imperative.